lnbits/db.py

```diff
@@ -388,9 +387,8 @@ class Page(BaseModel, Generic[T]):
 class Filter(BaseModel, Generic[TFilterModel]):
     field: str
     op: Operator = Operator.EQ
-    values: list[Any]
-
     model: Optional[type[TFilterModel]]
+    values: Optional[dict] = None
```

Comment:
This doesnt make sense as a dictionary since all the keys would be the same (`field`)

Reply:
it can be key value now, because placeholders can be `:key` not only `?`